### ISSUE TITLE
Anonymous users input date of birth

### ISSUE THREAD

**Issue by nagthanm** [fit-together-tv/ft-issue-report#13]
### Reproduction Steps
Anonymous Participants:
+ Open the exam link.
+ Input date of birth.

### Actual Behavior
+ First time participants may have some trouble to select year of birth although it took quite a while for me to figure out. But at the exam with not so much time, I suppose the participant do not have much calmnness to figure it out.

### Expect behavior
+ "Months" and "Years" on the first line should look more like something can be selectable.

### Device 
Windows 10, PC.
(Mobile UI is perfect)

### Browser
Chrome

https://drive.google.com/file/d/1kju0eOn13i7wUZRaMJQ7vg_1-M26GTzf/view?usp=sharing

**Comment by bugless-it** [comment]
@nagthanm I'm not able to open the videos. Do you have a screenshot?

**Comment by nagthanm** [comment]
Hi, did you try to open the file by main.chuongnau@gmail.com.
Because I only allow this email to access to the file.

**Comment by bugless-it** [comment]
I downloaded it to my laptop and tried opening using Windows Media Player, it didn't open unfortunately.

**Comment by bugless-it** [comment]
@nagthanm actually, i think you might be able to upload private videos to youtube

**Comment by nagthanm** [comment]
https://www.youtube.com/playlist?list=PLzzbcDv2nP-y8PXBCk-1x_XSpIuwDcmQq

@bugless-it I created a playlist here, please let me know if you can see it!

**Comment by nagthanm** [comment]
edited reupload

https://www.youtube.com/playlist?list=PLzzbcDv2nP-y8PXBCk-1x_XSpIuwDcmQq&jct=ciInnk5GIroHiC93UKY0dJX3zXiBCg

**Comment by nagthanm** [comment]
> I downloaded it to my laptop and tried opening using Windows Media Player, it didn't open unfortunately.

These video run ok on Windows Media Player on my PC???

**Comment by bugless-it** [comment]
@nagthanm i tried to access the link, it says playlist does not exist - are you able to give me access?

**Comment by bugless-it** [comment]
> > I downloaded it to my laptop and tried opening using Windows Media Player, it didn't open unfortunately.
> 
> These video run ok on Windows Media Player on my PC???

Let me try again. What media player were you using?

**Comment by bugless-it** [comment]
Ah ok, I'm able to open now. Actually the user can type their DoB in the input field, have you tried that approach? @nagthanm

**Comment by nagthanm** [comment]
I have 2 videos, the 2nd vid is shown I can select and choose the years. I have not try the input field. 
The first time I tried to input DoB, I'm a little panicked so I guess the participants either.

**Comment by nagthanm** [comment]
I will tell you once I try again

**Comment by nagthanm** [comment]
> @nagthanm i tried to access the link, it says playlist does not exist - are you able to give me access?
> 
> 
> 
> 

Let me try again, I set it 'private' and 'collaborate'

**Comment by nagthanm** [comment]
https://www.youtube.com/playlist?list=PLzzbcDv2nP-y8PXBCk-1x_XSpIuwDcmQq

https://www.youtube.com/playlist?list=PLzzbcDv2nP-y8PXBCk-1x_XSpIuwDcmQq&jct=5-5Wmv-kOuTUAEW89UoH9SGkWF47Hg

Hi, can you see the video?
I share it privately with main.chuongnau@gmail.com
@bugless-it

**Comment by bugless-it** [comment]
Awesome, I'm able to see it now @nagthanm

**Comment by nagthanm** [comment]
@bugless-it I suppose participants can type their DoB directly in the input field.
But when I select the DoB input field, there's a calendar pop up and cover almost all the input field. No matter what I scroll up or down, the calendar just stay in the header. Can you move the calendar to the right side next to the input field so people can see their DoB and they can confidently fill in their personal info?

![Input DoB](https://user-images.githubusercontent.com/87641533/129659637-bdd61afa-c418-4d5d-9754-a7617aff2729.png)

**Comment by bugless-it** [comment]
@nagthanm good suggestion, let me see how it can be done.

**Comment by bugless-it** [comment]
@nagthanm i think the above issue only happens if the user's screen is small in height. What's your screen dimension?

**Comment by nagthanm** [comment]
My PC screen resolution is 1366x768 @bugless-it

**Comment by bugless-it** [comment]
@nagthanm i updated the date/time pickers to use a simpler style. Can you try again?

**Comment by nagthanm** [comment]
Yes, I think the update is good. 
I will close this issue for now.
